src/components/manage/Blocks/Search/hocs/withSearch.jsx

```diff
@@ -333,7 +333,8 @@ const withSearch = (options) => (WrappedComponent) => {
       setSearchData(
         getInitialState(data, facets, urlSearchText, id, sortOn, sortOrder),
       );
-    }, [deepFacets, facets, data, urlSearchText, id, sortOn, sortOrder]);
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
maybe use JSON.parse(deepFacets) so you can avoid the eslint-disable line?

Reply:
it makes sense, I also have to add a deepData to remove the eslint-disable line.